### DISCUSSION TITLE
Create a GitHub personal token

### DISCUSSION THREAD

**Discussion by patsevanton** [lowlighter/metrics#394] [Q&A] [answered]
Hello! Thanks for metrics. 
It is great feature.
Did I fill in the fields correctly?
![Create a GitHub personal token](https://user-images.githubusercontent.com/10828883/122213313-a7de2000-ceca-11eb-86d2-bcc448673695.png)

**Comment by Zenoo** [comment, marked as answer]
If you want private repo metrics, you can also simply check the `repo` checkbox.

**Reply by patsevanton** [reply]
![Create a GitHub personal token1](https://user-images.githubusercontent.com/10828883/122213898-47031780-cecb-11eb-92fa-de82945c9207.png)


Is it necessary to specify all repo for private repo?

**Reply by Zenoo** [reply]
![](https://i.imgur.com/pN22zOL.png)

The whole scope seems to be needed here

**Reply by patsevanton** [reply]
Ок. Thanks!
